dataCenter/dataCenter.js

```diff
@@ -1,41 +1,104 @@
 const express = require('express');
 const app = express();
-const bodyParser =require('body-parser');
+const bodyParser = require('body-parser');
 const fs = require('fs');
-const cors = require('cors')
+const cors = require('cors');
+const cheerio = require('cheerio');
 
-app.use(cors())
+var http = require('http').Server(app);
+var io = require("socket.io").listen(http);
+
+
+io.sockets.on('connection',function (socket){
+  console.log("connection");
+
+socket.on("request", function (data){
+  data.forEach(obj => {
+    console.log (data.requestName + data.requestQuality + data.requestSecurity);
+  })
+});
+});
+
+app.use(cors());
 
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true }))
 
 const port = process.env.PORT || 3000;
 
+const resources = require('./resources');
+
+let remainingResources = resources;
+
 //this is express http request http post request can have information
 app.post('/dc1', (req,res) => {
```

Comment:
`/dc1` endpoint will probably have to be dynamically set using an env variable/parameter that we pass in when starting up this service. (i.e `process.env.DOMAIN`)